06-Plotting-Time-Series-Rasters-in-R.Rmd

```diff
@@ -171,27 +177,24 @@ levelplot(NDVI_stack,
 
 ```
 
-Now we have a nicer looking plot, with colors that make more sense given the 
-data we are working with. But the labels for each raster are a bit clunky. 
-Let's make sure each label represents the Julian day for the data. The names
-come from the bands or layers in the `RasterStack`.
+For all of the `brewer.pal` ramp names see the <a href="http://www.datavis.ca/sasmac/brewerpal.html" target="_blank"> brewerpal page</a>.
 
-##Refining labels in our plot 
+Cynthia Brewer, the creater of ColorBrewer, offers an online tool to help choose suitable color ramps, or to create your own. <a href="http://colorbrewer2.org/" target="_blank">ColorBrewer 2.0; Color Advise for Cartography </a>
 
-The first part of the name is the Julian Day. We might want to remove the "x" and 
-replace it with Julian Day. We might want to remove the  the `_HARV_ndvi_crop` 
-from each label and replace it with the YEAR. We can replace text with different 
-text using the `gsub` function in R as follows: 
+The yellow to green color ramp make more sense given the data we are working
```

Comment:
The yellow to green color ramp visually represents NDVI well given it's a measure of greenness. Someone looking at the plot can quickly understand that pixels that are more green, have a higher NDVI value.